packages/csv-parser-price/src/main.js

```diff
@@ -183,9 +183,11 @@ export default class CsvParserPrice {
 // Easiest way to wrap the getCustomTypeDefinition in the memoize method
 CsvParserPrice.prototype.getCustomTypeDefinition = memoize(
   function _getCustomTypeDefinition (customTypeKey) {
-    const getTypeByKeyUri = createRequestBuilder().types
+    const getTypeByKeyUri = createRequestBuilder({
```

Comment:
This module [uses](https://github.com/commercetools/nodejs/blob/master/packages/csv-parser-price/src/main.js#L13) the `createRequestBuilder()` method, which is part of the `api-request-builder`, which is this PR